src/header/common/content_type.rs

```diff
@@ -74,6 +74,12 @@ impl ContentType {
         ContentType(mime::TEXT_HTML)
     }
 
+    /// A constructor  to easily create a `Content-Type: text/xml; charset=utf-8` header.
```

Comment:
The docs say `text/xml; charset=utf-8`, but [`TEXT_XML`](https://docs.rs/mime/0.3.4/mime/constant.TEXT_XML.html) is `text/xml`. Either the documentation is wrong, or the implementation :D

I've typically stayed away from XML as much as possible (lifestyle choice), so I can't say from experience which is more common. I suspect including the charset is?

Reply:
Hehe; for me its more without charset; made a fix in https://github.com/spk/hyper/commit/4d893a7fc9cf2b187598298dc467b4db9b44d516